pyluna-pathology/luna/pathology/cli/dsa/dsa_annotations.py

```diff
@@ -316,13 +193,20 @@ def generate_geojson(
         metadata (Dict[str, any]): slide metadata
         labelset (str): name of the labelset
         slide_store_dir (str): filepath to slide datastore
+        annotation_type (str): the type of annotation to pull from DSA, cooresponding to either
```

Comment:
Should be 'data_type'? It would be nice to document the expected values like "REGIONAL_METADATA_RESULTS" or "POINT_GEOJSON"

Reply:
So, from the `data_type`, which is a fixed set of values from [constants](https://github.com/msk-mind/luna/blob/dev/pyluna-common/luna/common/constants.py), I 'infer' the annotation_type (regional or point). I'll document the expected values.

Reply:
yea, adding a validation check like L269-272, with an improved error message would be great also. 
'Invalid annotation type: {data_type}. Expected data types are "REGIONAL_METADATA_RESULTS" or "POINT_GEOJSON"'